lib/restful/jsonapi/version.rb

```diff
@@ -1,5 +1,5 @@
 module Restful
   module Jsonapi
-    VERSION = "0.0.13"
+    VERSION = "1.0.0"
```

Comment:
This should be our first major release. According to semver website:

http://semver.org/

>How do I know when to release 1.0.0?
>
>If your software is being used in production, it should probably already be 1.0.0. If you have a stable API on which users have come to depend, you should be 1.0.0. If you’re worrying a lot about backwards compatibility, you should probably already be 1.0.0.